app/views/layouts/angular/_multi_auth_credentials.html.haml

```diff
@@ -188,13 +188,59 @@
                                     :id                => record.id,
                                     :prefix            => "amqp",
                                     :basic_info_needed => true}
+
```

Comment:
Do you think it would make sense to merge this part L191-231 with OpenStack? We now have three options (none, ceilometer and amqp) but the two sections are pretty much duplicated.